src/flt-player.c

```diff
@@ -223,7 +223,7 @@ static struct player_pattern *player_pattern_new (struct player_pattern *src) {
 
 static void pattern_clist_sync_selection (void) {
 	GSList *list;
-	struct player_pattern *pp;
+	struct player_pattern *pp = {};
```

Comment:
It seems like this should be set to NULL as it's a pointer.

Reply:
Ah yes. Thanks.

Reply:
Fixed.